### ISSUE TITLE
Support for logout

### ISSUE THREAD

**Issue by nicosabena** [auth0/custom-social-connections#32]
Add support for logout in the UI. `oauth2` connections support two options for logout:

- `logoutUrl` that holds a string value with a fixed logout URL to redirect the user to when a federated logout is requested.
- `getLogoutUrl`, a script that can generate a logout URL dynamically based on the request query. The script format is like this:

```js
function(query, callback) {
  var logoutUrl = "...";
  callback(null, logoutUrl);
}
```

**Comment by mleybsage** [comment]
@nicosabena in the linked article https://community.auth0.com/t/custom-social-connection-federated-logout/19491 you guys mention that 
"if you use the UI after setting any of these options and click Save, the custom property you set before will be lost."
is this behaviour intentional? We've recently been affected by the changed we've applied to the options object being lost when interaction through the UI occurs

**Comment by nicosabena** [comment]
> is this behaviour intentional?

No, not really. It's just that at the moment this extension was created there was no need to handle the scenario where certain properties like `logoutUrl ` (not available in this extension's UI) were modified directly using the API. 
This extension recreates the connection when clicking "Save" based on whatever is seen in the UI, so any properties that the extension doesn't know about are lost. Not ideal, of course, and something that could be improved.

**Comment by BryceBarbara** [comment]
@nicosabena Is it possible to prevent modifications to a social connection if it has those unsupported properties in it? We'd like to avoid scenarios where it can be broken by absent minded developers/admins.

**Comment by nicosabena** [comment]
Custom Social Connections are a first-class citizen in the Auth0 Dashboard now, so there's no need to use the Custom Social Connection Extension.

This appears at the bottom of the list when creating a new social connection:

<img width="338" alt="image" src="https://github.com/auth0/custom-social-connections/assets/5222181/dec3faae-f4f2-42f8-8138-ff371c70d73f">

And when editing one of these connections, you get the usual fields:

<img width="1035" alt="image" src="https://github.com/auth0/custom-social-connections/assets/5222181/653e6ce5-ffd5-4ceb-b9f8-a2c63a8678c7">

`logoutUrl` and `getLogoutUrl` aren't directly available (as before), but the good news is that the dashboard UI will not overwrite existing values for the connections, so if you set either value using the Management API and then edit the connection from the dashboard the logout configuration will stay.

So, TL;DR: stop using the Custom Social Connection extension and use the Auth0 Dashboard directly to manage custom social connections 👍
